src/Microsoft.Docs.Template/Models/LandingData.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace Microsoft.Docs.Build
 {
-    [PageSchema]
+    [PageSchema(contributors: false)]
```

Comment:
>false [](start = 30, length = 5)

they are not controlled by configuration?

Reply:
by both.

---
In reply to: [213583452](https://github.com/dotnet/docfx/pull/3316#discussion_r213583452) [](ancestors = 213583452)

Reply:
hmmm, i don't like two source of truth


---
In reply to: [213583681](https://github.com/dotnet/docfx/pull/3316#discussion_r213583681) [](ancestors = 213583681,213583452)

Reply:
I am thinking maybe this can more clear for output defination:

PageSchema(Ignore: output=> output.Contributors)

---
In reply to: [213584156](https://github.com/dotnet/docfx/pull/3316#discussion_r213584156) [](ancestors = 213584156,213583681,213583452)